JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+msgstr "fout % >"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "R-hat >"
+msgstr "R-hat >"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimated sample size <"
+msgstr "Geschatte steekproefgrootte <"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Redistribute prior probability"
+msgstr "Herverdeel prior kans"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Conditional models"
+msgstr "Conditioneel modellen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Model space"
+msgstr "Modelruimte"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Control"
+msgstr "Controle"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Clever refitting"
+msgstr "Slim opnieuw passen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Do not refit"
+msgstr "Pas het niet opnieuw"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Always refit"
+msgstr "Pas het altijd opnieuw"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Save the fitted model"
+msgstr "Sla de gefitte model op"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Dependent variable"
+msgstr "Afhankelijke variabele"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Number of trials"
+msgstr "Aantal proeven"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Fixed effects variables"
+msgstr "Vaste-effecten variabelen"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Random effects grouping factors"
+msgstr "Willekeurige effect grouperingsfactoren"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Family"
+msgstr "Familie"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Link"
+msgstr "Koppeling"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Logit"
+msgstr "Logit"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Probit"
+msgstr "Probit"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Cauchit"
+msgstr "Cauchit"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Complementary LogLog"
+msgstr "Complementair LogLog"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Identity"
+msgstr "Identiteit"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Log"
+msgstr "Log"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Sqrt"
+msgstr "Wortel"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Inverse"
+msgstr "Inverse"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Estimated marginal means"
+msgstr "Geschatte marginale gemiddelden"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Model variables"
+msgstr "Model variabelen"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Selected variables"
+msgstr "Geselecteerde variabelen"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Response scale"
+msgstr "Continue uitkomstmaat"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Specify contrasts"
+msgstr "Specificeer contrasten"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Estimated trends/conditional slopes"
+msgstr "Geschatte trends/conditionele hellingen"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Continous variables"
+msgstr "Continue variabelen"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Trend variable"
+msgstr "Trend variabele"
+
+msgctxt "MixedModelsBLMM|"
+msgid "Dependent variable"
+msgstr "Afhankelijke variabele"
+
+msgctxt "MixedModelsBLMM|"
+msgid "Fixed effects variables"
+msgstr "Vaste effect variabelen"
```

Comment:
Zelfde bij deze. Gefixeerd of vast? Sowieso streepje ertussen als we voor vast gaan, zodat deze zin hetzelfde is als de andere